ModularBungalow/clothing/neocoat.dm

```diff
@@ -80,58 +80,3 @@
 	icon_state = "labcoat_geo"
 	allowed = list(/obj/item/pickaxe, /obj/item/tank/internals)
 	armor = list(MELEE = 25, BULLET = 0, LASER = 10,ENERGY = 10, BOMB = 30, BIO = 0, RAD = 0, FIRE = 100, ACID = 50)
-
-
-
-//Trench labcoats
```

Comment:
Voted against by the community.

Reply:
The point of the longcoat resprite was supposed to replace the old ones, not add an alternative. If youre just going to keep the old one anyways, then what's the point of a resprite?

Reply:
V a r i e t y

plus, it lets people choose whatever they want. This is why so many people hate combat mode, and one of the reasons we get new players. We try not to force new things down your throat and let you choose what you prefer. This carries over to fashion.

Reply:
Its resprite, not adding a new uniform

Reply:
But people want it to be a new uniform. So you should.

Reply:
> Its resprite, not adding a new uniform

If the community says no, That means no. Going around to make another PR to try and circumvent proper opinion is disrespectful. 
It doesn't matter what it was intended as, 
**This server is a democracy.**